src/components/Editor/AdditionalInformation/AccessPoints.vue

```diff
@@ -0,0 +1,247 @@
+              <v-container fluid>
+                <v-row justify="start">
+                  <v-col class="col-6">
+                    <v-select
+                      v-model="newAccessPoint.type"
+                      :items="availableTypes"
+                      menu-props="auto"
+                      label="Type"
+                      :rules="[rules.isRequired()]"
+                      outlined
+                      return-object
+                    />
+                  </v-col>
+                  <v-col class="col-6">
+                    <v-text-field
+                      v-model="newAccessPoint.url"
+                      label="URL"
+                      :rules="[rules.isRequired(), rules.isUrl()]"
+                      outlined
+                    />
+                  </v-col>
+                  <v-col class="col-6">
+                    <v-text-field
+                      v-model="newAccessPoint.documentation_url"
+                      label="Documentation URL"
+                      :rules="[rules.isUrl()]"
+                      outlined
+                    />
+                  </v-col>
+                  <v-col class="col-6">
+                    <v-text-field
+                      v-model="newAccessPoint.example_url"
+                      label="Example URL"
+                      :rules="[rules.isUrl()]"
+                      outlined
+                    />
+                  </v-col>
+                </v-row>
+              </v-container>
+            </v-card-text>
+            <v-card-actions>
+              <v-btn
+                class="green white--text"
+                @click="addAccessPoint()"
+              >
+                Add Access Point
+              </v-btn>
+              <v-btn
+                class="red white--text"
+                @click="openEditor = false"
+              >
+                Cancel
+              </v-btn>
+            </v-card-actions>
+          </v-card>
+
+          <!-- new access point dialogue ends -->
+        </v-row>
+      </v-container>
+    </v-dialog>
+  </v-card>
+</template>
+
+<script>
+import {mapGetters} from "vuex"
+//import {mapGetters, mapState} from "vuex"
+import {isRequired, isUrl} from "@/utils/rules.js"
+
+
+export default {
+  name: "AccessPoints",
+  data() {
+    return {
+      openEditor: false,
+      newAccessPoint: {},
+      rules: {
+        isRequired: function(){return isRequired()},
+        isUrl: function(){return isUrl()},
+      },
+      availableTypes: ["REST", "SOAP", "SPARQL", "Other"]
+    }
+  },
+  computed: {
+    ...mapGetters("record", ["getSection", "getChanges"]),
+    section(){
+      return this.getSection('additionalInformation');
+    },
+    initialFields(){
+      return this.getSection("additionalInformation").initialData
+    },
+    currentFields(){
+      let data = this.getSection("additionalInformation").data
+      if (!data.access_points) {
+        data.access_points = [];
+      }
```

Comment:
```suggestion
      if (!data.access_points) data.access_points = [];
```
In JS you are allowed to write single statement of an IF/Else on the same line without the curly brackets.